src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,61 @@
+import { ReactPlugin, withAITracking } from '@microsoft/applicationinsights-react-js';
+import { ApplicationAnalytics, ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ComponentType } from 'react';
+import ITelemetry from './ITelemetry';
+
+class Telemetry implements ITelemetry {
+  private appInsights: ApplicationInsights;
+  private config: any;
+
+  constructor() {
+    const areWeInDev = process.env.NODE_ENV === 'development';
```

Comment:
[nit pick]
weAreIndev sounds more confident than areWeInDev :-)

Reply:
Used `areWeInDev` because of how the variable is used later on `areWeInDev ? false : true`